Casks/p/pixpin.rb

```diff
@@ -2,14 +2,14 @@
   version "1.9.5.0"
   sha256 "30e43609473a4894d880c8c0774f43c1d91497df9639c0e72ceef7d9ca275970"
 
-  url "https://download.pixpinapp.com/PixPin_#{version}.dmg"
+  url "https://download.pixpin.cn/PixPin_#{version}.dmg"
   name "PixPin"
   desc "Screenshot tool"
-  homepage "https://pixpinapp.com/"
+  homepage "https://pixpin.cn/"
 
   livecheck do
     url :homepage
-    regex(/href=.*?PixPin[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
+    regex(%r{href="/docs/official-log/v?(\d+(?:\.\d+)+)}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/docs/official-log/v?(\d+(?:\.\d+)+)(?:\.html)?["' >]}i)
```

We use the `href=.*?` pattern because it accounts for variations in attribute delimiters (`"`, `'`, or nothing) and will continue to match if the URL becomes absolute instead of relative.

There's something to be said for anchoring the end of the version (using `["' >]` in this context) to reduce the chance of this matching something unexpected like an unstable version. The links in the sidebar of the docs pages use a `.html` extension, so we'll need to account for that possibility.